soccer/gameplay/generate_fsm_diagrams.py

```diff
@@ -21,8 +20,20 @@ def mkdir_p(path):
 logging.getLogger().setLevel(logging.INFO)
 
 import main
+import ui.main
 main.init()
 
+class moc_Point():
```

Comment:
Unfortunately we can't do this (at least with our current setup, which is the root of this problem in the first place). The generate-behavior-diagrams script runs python on these files directly (which means it's not stated via boost python, and the entire python-cpp bridge is unavailable). 

I was thinking about trying to get a cpp target that we could call (and start python and run the behhavior diagram scripts) which would allow us to get everything we need, but I don't think thats feasable in the short term (although it would be a good intermediate project for those who want to learn about the python bridge). I could be overlooking something, so let me know if I'm just being stupid :P